archinstall/lib/simple_menu.py

```diff
@@ -76,7 +76,7 @@
 DEFAULT_CYCLE_CURSOR = True
 DEFAULT_EXIT_ON_SHORTCUT = True
 DEFAULT_MENU_CURSOR = "> "
-DEFAULT_MENU_CURSOR_STYLE = ("fg_red", "bold")
```

Comment:
Can we leave the actual simple_menu file as is and edit this on the caller side? 
There's already a a variable defined there to override the cursor style 
```		
main_menu_cursor_style = ("fg_cyan", "bold")
```